(base) F:\UW TACOMA\TCSS 555 - Machine learning\Final Project\tcss555\MalwareDetection>python malware.py malware_train_partial2.csv
------ Start of training on sample data --------
Started Cleanup: 2019-03-16 03:35:25.250799
Number of rows in data [Before Data Cleaning - But converting NAN/INF]: 1499999
Ending Cleanup At: 2019-03-16 03:35:50.986667
Starting Convertion at: 2019-03-16 03:35:50.986667
C:\Users\Unknown_\Anaconda3\lib\site-packages\numpy\core\fromnumeric.py:83: RuntimeWarning: overflow encountered in reduce
  return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
Finishing Convertion at: 2019-03-16 03:35:59.127840
STACK LAYER #1: First Classifier (RandomForests) in STACK is processing ...
Start Modeling with model 1: 2019-03-16 03:35:59.127840
Accuracy M1: 61.16%
Accuracy M1: 61.15%
Accuracy M1: 61.70%
Accuracy M1: 57.25%
Accuracy M1: 59.85%
Finished Modeling with model 1: 2019-03-16 03:42:38.231562
STACK LAYER #1: First Classifier (RandomForests) in STACK is Finished successfully.
STACK LAYER #2: Second Classifier (Neural Network) in STACK is processing ...
Start Modeling with model 2: 2019-03-16 03:42:38.231562
Accuracy M2: 62.68%
Accuracy M2: 62.15%
Accuracy M2: 62.38%
Accuracy M2: 61.92%
Accuracy M2: 60.39%
Finish Modeling with model 2: 2019-03-16 04:04:57.493494
STACK LAYER #2: Second Classifier (Neural Network) in STACK is finished successfully.
STACK LAYER #3: Third Classifier (SVM) in STACK is processing ...
Start Modeling with model 3: 2019-03-16 04:04:57.493494
Accuracy M3: 50.00%
Finish Modeling with model 3: 2019-03-16 03:34:28.265155
STACK LAYER #3: Third Classifier (SVM) in STACK is finished successfully.
Finiahed At: 2019-03-16 03:34:28.265155
------ End of training on sample data --------

(base) F:\UW TACOMA\TCSS 555 - Machine learning\Final Project\tcss555\MalwareDetection>python malware-auc.py malwaremodel.pkl malware_test.csv
--------------------------------- Start predict on TEST data---------------------------------
Number of rows in data [Before Data Cleaning - But converting NAN/INF]: 30
C:\Users\Unknown_\Anaconda3\lib\site-packages\pandas\core\generic.py:6130: SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy
  self._update_inplace(new_data)
malware-auc.py:457: SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy
malware-auc.py:459: SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy
malware-auc.py:470: SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy
malware-auc.py:471: SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy
malware-auc.py:472: DataConversionWarning: Data with input dtype int64, float16, int32, float64 were all converted to float64 by StandardScaler.
test data dim: 30
finished populating predicted result in M1.
finished populating predicted result in M2.
finished populating predicted result in Final level of Stack.
--------------------------------- End of predict on TEST data---------------------------------
AUC: 0.70



